pkg/transport/transport.go

```diff
@@ -104,7 +111,13 @@ func (transport *Transport) DialAddress(ctx context.Context, address string, opt
 		grpc.WithBlock(),
 		grpc.FailOnNonTempDialError(true),
 		grpc.WithUnaryInterceptor(InvokeTimeout{transport.requestTimeout}.Intercept),
-		grpc.WithStreamInterceptor(InvokeStreamTimeout{transport.requestTimeout}.Intercept),
+		grpc.WithContextDialer(func(ctx context.Context, addr string) (net.Conn, error) {
+			conn, err := (&net.Dialer{}).DialContext(ctx, "tcp", addr)
```

Comment:
would it be a benefit to have a `const TCP = "tcp"`

Reply:
To me this seems more weird. I'm expecting to see "tcp", so seeing a constant forces me to lookup what the constant value is...